### DISCUSSION TITLE
Create multi tag and several solutions

### DISCUSSION THREAD

**Discussion by beatific-angel** [InnovateAsterisk/Browser-Phone#297] [General]
Dear @InnovateAsterisk 
your solution is very useful and great.
I have some questions.
currently this is second time to using this solution
but whenever use this solution, I have to modify your code for many parts...

So I would like to divide this solution according to several tags, it will be very useful for people who use it.
If you like this and if you don't have the time, I'll be happy to group your results according to different tags.

For instance, some users may not allow video calls.
And some users may want asterisk + freepbx + webrtc solution.
And some users may want ubuntu or centos solution.
It would be very useful to classify them accordingly.

**Comment by InnovateAsterisk** [comment]
What are the kind of changes do you need to make?



> For instance, some users may not allow video calls.

There is an option for this: https://github.com/InnovateAsterisk/Browser-Phone/blob/6f5051f5bc90b9927d06e38f00a6bec3061b93fd/Phone/phone.js#L144

> And some users may want asterisk + freepbx + webrtc solution.

The Browser Phone works the same with this configuration.

> And some users may want ubuntu or centos solution.

The Browser Phone works the same with these operating systems: https://www.innovateasterisk.com/s2e2-webrtc-in-the-cloud/

**Comment by beatific-angel** [comment]
yes. it is good.
that is very useful.
It is very inconvenient for multiple users to inquire about this and you may not have the time.
So I think it would be very useful to tag these things.
